dev-docs/howtoguides/how_to_use-magic_attach_endpoints.md

```diff
@@ -0,0 +1,165 @@
+# How to use magic attach API endpoints
+
+Right now, the pro client provides three distinct endpoints to make it easier to perform
+the magic attach flow. They are:
+
+* u.pro.attach.magic.initiate.v1
+* u.pro.attach.magic.wait.v1
+* u.pro.attach.magic.revoke.v1
+
+We will explain how to use each endpoint and what is the expected output for each of them.
+
+## Initiate endpoint
+
+To start the magic attach flow, we need to create a token for it. The initiate endpoint
+will perform exactly that. When you run:
+
+```console
+$ pro api u.pro.attach.magic.initiate.v1
+```
+
+It is expected for you to see the following json response:
+
+```json
+{
+  "_schema_version": "v1",
+  "data": {
+    "meta": {
+       "environment_vars": []}
+    },
+    "attributes": {
+      "expires": "EXPIRE_DATE",
+      "expires_in": SECONDS_TO_EXPIRE, 
+      "token": "MAGIC_ATTACH_TOKEN",
+      "user_code": "USER_CODE"
+    }
+    "type": "MagicAttachInitiate"
+  },
```

Comment:
A small linting error. I think there is some json linting error.

Reply:
Good catch. I have just fixed it